Stories/Readme.md

```diff
@@ -0,0 +1,52 @@
+# Welcome and Thanks!
+
+We are trying to develop an online data analysis game for citizens. The goal is to give citizens the opportunity to learn more about DNA sequencing and microbiome data. 
+To make it more fun we want to tell the game in a story. We have a few story ideas but we are not sure which is the most capturing. These short stories should get people interested in the game. It would be awesome if you could give us feedback on the stories below. 
+
+
+
+### Story 1: Alien discovering the earth
+
+- [Google Slides](https://docs.google.com/presentation/d/1rhOmXrTEEv7h4ECyFz-zGUFNusMItufe2GqqXX1qySQ/edit?usp=sharing)
+
+During a space exploration, Sam, a curious young alien robot from a faraway galaxy, discovered a little blue planet in the Milky Way. As he approached it, he saw that it was water on top of a rocky planet with an atmosphere of nitrogen and oxygen. But wait, what are these little creatures everywhere? In the water, on the rocks and even flying in the atmosphere! In different shapes and sizes and they are even creating replicates of themselves! Is this called life? Sam had heard of life before but he thought it was a myth.
+Sam wanted to take different specimens home for his terrarium and to show their robot friends, but these fragile life creatures wouldn't make it through the wormholes intact. Sam decided to examine and study how they function so they could build his own self-replicating creatures on his home planet.
+Can you help Sam to decipher the DNA?
+
+
+
+### Story 2: Beer lover Anna wants to investigate craft beers 
+
+- [Google Slides](https://docs.google.com/presentation/d/1FeZkuDztakgAw-oHA6b-mFGYFs3GdNnO3eRAybJGHDU/edit#slide=id.gcf3936e2dc_0_1)
+
+Meet Anna! She is an architect who plays volleyball, likes to go hiking, loves reading and drinking beer. One day she came across an article discussing the beer microbiome.
+This got her wondering what is a beer microbiome? There are collections of small living creatures. Therefore she starts researching. She finds out that these small creatures are called bacteria and they are everywhere. In our gut, in the soil, even on vending machines. Some of these bacteria are actually very good for us. And some others can make us very ill. Bacteria come in different shapes and sizes but they have the same components. One crucial component is the DNA, the blueprint of life. The DNA encodes the shape and size and many other things unique for a bacterial species. Because of the encoding information the DNA can be used to identify what kind of bacteria the DNA is from. Therefore within a sample form soil, our gut or beer one can specify what kind of species are inside the sample. Anna would like to learn more about this kind of analysis. Can you help Anna to find out more?
+
+
+
+### Story 3: wine makers: Jeff wants to decode an old family wine recipe
```

Comment:
in this story, the goal is not to analyse complete microbiome as for the second story, but searching a specific species on the sample and determine the precise variety. Just to comment the fact that this is not completely the same question / data anlaysis workflow you need depending on the story we are looking for, and this can have important repercussions 1/  on the Galaxy side for example (on the tools and workflows you need) + 2/ on the "overall" topic (as mentionned before, is microbiome DNA analysis THE overall goal or maybe something broader like "DNA analysis" ?)

Reply:
In beer and wine, the living organisms left we can detect by sequencing are just the yeasts, I think. If this is the case the data analysis would be quite similar to the one we did in the BeerDEcoded project. Maybe I miss something here?
But of course, the negative point here is once jeff finds the Yeasts his initial search is over. Maybe here Jeff can find in the first and second level his family recipe but then encounters that finding out about specicies in a sample is very exciting. Maybe the influence of alcohol on the gut microbiome? I will work on this.